src/messages/util.js

```diff
@@ -34,10 +34,7 @@ function convertHighlightToSlackMarkup(highlight) {
 
 function extractKnowledgeStoreHighlight(item) {
   if (item.captions && item.captions.length > 0) {
-    return (
-      convertHighlightToSlackMarkup(item.captions[0].highlights) ||
-      "Generated highlight not available at this time"
-    );
+    return convertHighlightToSlackMarkup(item.captions[0].highlights) || " ";
```

Comment:
something like this would be what I was trying to explain yesterday
```suggestion
  const highlights = item.captions[0].highlights
  if (highlights?.length > 0) {
    return highlights
  }
  
  if (item?.captions[0]?.text?.length > 0) {
    return item.captions[0].text
  }
   // fallback to returning a single space if nothing else, (slack won't allow an empty block here)
   // TODO refactor so that the block isn't added if empty
    return " ";
```

concisely this may work as well but i'm not 100% sure on it:

```suggestion
    return convertHighlightToSlackMarkup(item.captions[0].highlights) || item.captions[0].text || " ";
```